redash/destinations/pagerduty.py

```diff
@@ -0,0 +1,69 @@
+import pypd
```

Comment:
This should be in a try/catch block and should set a boolean value which used in the `enabled` method of the destination like we do for data sources.